src/plans/details/tabs/Details/components/SettingsSection/components/RootDisk/EditRootDisk.tsx

```diff
@@ -31,18 +28,19 @@ const EditRootDisk: FC<EditPlanProps> = ({ resource }) => {
       <Stack hasGutter>
         <EditRootDiskModalBody />
         <EditRootDiskModalAlert vms={vms} />
-        <FormGroup label={t('Root device')}>
-          <FilterableSelect
-            selectOptions={dropdownItems}
+        <FormGroupWithHelpText
+          label={t('Root device')}
+          helperText={t(
+            'Provide the storage device or partition that contains the root filesystem. If left blank, the first root device will be used.',
+          )}
+        >
+          <TextInput
             value={value}
-            onSelect={(newValue) => {
-              setValue(newValue as string);
+            onChange={(_event, newValue) => {
+              setValue(newValue);
             }}
-            canCreate
-            placeholder={t('First root device')}
-            createNewOptionLabel={t('Custom path:')}
           />
-        </FormGroup>
+        </FormGroupWithHelpText>
```

Comment:
same here..worth adding validations for the text field before applying, specifically if an invalid value might fail the migration (helperTextInvalid, validated..)